pages/blog/[slug].tsx

```diff
@@ -37,10 +37,10 @@ const components = {
       allowFullScreen={true}
     ></iframe>
   ),
-  CreateAppCta: ({ ctaText, cliText }) => (
+  CreateAppCta: ({ ctaText,ctaURL, cliText }) => (
```

Comment:
If the URL is configurable, should this be renamed to something more generic?
(E.g CTAButton)

Reply:
I add it to be config so that we don't have to manually update when we update the docs URL in X months time

Reply:
> I add it to be config so that we don't have to manually update when we update the docs URL in X months time

I'm not sure what you mean here. If we change the docs URL, wouldn't that mean needing to change every instance of CreateAppCta instead of changing the one link within `CreateAppCta`? 

Not a big deal though, but just felt odd that the name `CreateAppCta` feels very specific to "Creating an app", however, the link is configurable.

Reply:
If you want to keep it configurable though, you'll also have to set it on the other places we use `CreateAppCta`